nnpdf_data/nnpdf_data/new_commondata/ATLAS_TTBAR_8TEV_LJ_DIF/filter.py

```diff
@@ -444,4 +446,12 @@ def processData():
     with open('uncertainties_dSig_dyttBar_norm.yaml', 'w') as file:
         yaml.dump(uncertainties_dSig_dyttBar_norm_yaml, file, sort_keys=False)
 
+def remove_commas():
+    pattern = "uncertainties*.yaml"
+    reg = re.compile(fr'({"sys,"})')
+    for file in Path(".").glob(pattern):
+        new_text = reg.sub("syst_", file.read_text())
+        file.write_text(new_text)
+
 processData()
+remove_commas()
```

Comment:
Could you run black black/isort again on these files?

Reply:
is it fine if I run black and isort on the entire new commondata itself? Because I see that when I do run them, they end of changing reformatting almost every new data implementation filter.

Reply:
Since no-one else is touching those files now, so you may as well. Makes life easier when reviewing those later since it keeps the diffs cleaner.

If that had not been the case, something like this will allow you to just run it on the files changed in a given branch `git diff --name-only origin/HEAD | xargs black`